addons/mail/static/src/components/thread_view/thread_view.js

```diff
@@ -68,6 +77,17 @@ export class ThreadView extends Component {
     // Handlers
     //--------------------------------------------------------------------------
 
+    _onClickCaptureGlobal(ev) {
+        if (this.threadView.messageReactionGroup &&
+            this.threadView.messageReactionGroup.message.messageReactionsSummary.component &&
+            this.threadView.messageReactionGroup.message.messageReactionsSummary.component.el.contains(ev.target)) {
+            return;
+        }
+        this.threadView.threadViewer.update({
+            hasReactionsSummary: false,
+        });
+    }
```

Comment:
(I didn't check the need for this but) move handler into corresponding model

Reply:
I'm not sure if I can move this in model....